### DISCUSSION TITLE
I can't approve my environment workflow

### DISCUSSION THREAD

**Discussion by metaclips** [community/community#39876] [Actions]
### Select Topic Area

Bug

### Body

I am running a workflow behind an environment, the environment requires that workflow be approved by an administrator (which I am), currently, starting a workflow myself doesn't show the button to review and approve deployment. Can an administrator approve their own workflows (I recently do approve workflows I start and I'm not sure if something changed in GitHub rules).

**Comment by ossanna16** [comment]
Hi there @metaclips and welcome to our community! Thank you for asking a great question 🙂
While you are waiting for a reply to your question, here are a few things you can do:

To get started, introduce yourself in our [official introduction thread](https://github.com/orgs/community/discussions/23204)
To connect with other new users, check out this Discussion to [Meet Your GitHub Community Team](https://github.com/orgs/community/discussions/28776), further your GitHub knowledge by working through our [GitHub Skill 🆙 exercises](https://github.com/orgs/community/discussions/32121), and participate in our [weekly Fun Friday chats](https://github.com/orgs/community/discussions/30912)
